dark/index.html

```diff
@@ -69,13 +69,13 @@
 
     <ul class="nav-links">
       <li>
-        <a href="#home" class="nav-items" id="home" onclick="home()">home</a>
+        <a href="#home" class="nav-items" onclick="home()">home</a>

```

Comment:
Add it like this in every link and change it in the js file also `assets/js/app.js`
```suggestion
        <a href="#home" class="nav-items" id="home-link" onclick="home()">home</a>
```

Reply:
ok working on it

Reply:
Sure